remappers/const.go

```diff
@@ -0,0 +1,27 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package remappers
+
+const (
+	// OTelTranslatedLabel is added as an attribute to each datapoint that
+	// is translated from otel.
+	OTelTranslatedLabel = "otel_translated"
```

Comment:
[**For reviewers**] This label is added to all data points that are created using the remapping logic. Suggestions for the label name is welcome if the current one doesn't look good.

In a followup PR, I plan to add the same label to the source metric's scope attributes and filter the attribute to avoid double translation.